projects/1.20/assets/complicated-bees/complicated_bees/lang/zh_cn.json

```
@@ -46,6 +46,7 @@
   "block.complicated_bees.apiary": "蜂房",
   "block.complicated_bees.centrifuge": "离心机",
   "block.complicated_bees.generator": "发电机",
+  "block.complicated_bees.apid_library": "蜜蜂图书馆",

```

单方块的“图书馆”🤔

---

其实本来是打算用“数据库”的，但毕竟和林业里的Database原文不同
```suggestion
  "block.complicated_bees.apid_library": "蜜蜂文库",
```